todo_app/app.py

```diff
@@ -1,27 +1,31 @@
 
-from todo_app.data.trello_board import *
+from todo_app.data.todo_mongo_client import *
 from todo_app.data.viewmodel import *
 from flask import Flask, render_template, request, redirect
+import os
 
-def create_app():
+def create_app(db_name = ""):
```

Comment:
Here you allow the user to pass in a database name, but in most cases you just read from an environment variable. Is there any advantage to having this over just asking your end-to-end test to set the environment variable like it did with the Trello board?